lib/webauth.js

```diff
@@ -39,10 +40,15 @@ function getAuthPluginSession(req, pluginID, dflt) {
 
 function setAuthPluginSession(req, pluginID, authPluginSession) {
   if (req.session) {
-    // FIXME Note that it does something only when req.session[pluginID] is 
+    // FIXME Note that it does something only when req.session.authPlugins[pluginID] is 
```

Comment:
so I used to simply divide the session by having `session["com.example.plugin1"]`, `session["com.example.plugin2"]`, etc, where each of those objects is the plugin's private storage. Now we have `session.authPlugins["com.example.plugin1"]` instead because we also have `session.zluxExp` right?

I'd say that, first, we might want to group all framework data also under a separate object in the session, and second, maybe that name can be a little more descriptive, like, `session.zlux.expirationTime`?

Reply:
zluxExp, but also because session.id existed, so previously it was technically true that a plugin could not be named "id". Not sure what wins the race. Sure, we can do zlux & zlux.expirationTime.